src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -525,6 +526,400 @@ public void changedDueDateAfterRecallingAnItemShouldRespectTenantTimezone()
       storedLoan.getString("dueDate"), isEquivalentTo(expectedDueDate));
   }
 
+  @Test
+  public void pagedItemRecalledThenLoanedAndNextRecallDoesNotChangeDueDate()
+      throws InterruptedException,
+      ExecutionException,
+      TimeoutException,
+      MalformedURLException {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestServicePoint = servicePointsFixture.cd1();
+    final IndividualResource jessica = usersFixture.jessica();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource james = usersFixture.james();
+
+    final LoanPolicyBuilder canCirculateRollingPolicy = new LoanPolicyBuilder()
+        .withName("Can Circulate Rolling With Recalls")
+        .withDescription("Can circulate item With Recalls")
+        .rolling(Period.weeks(3))
+        .unlimitedRenewals()
+        .renewFromSystemDate()
+        .withRecallsMinimumGuaranteedLoanPeriod(Period.weeks(2))
+        .withRecallsRecallReturnInterval(Period.weeks(1));
+
+    final IndividualResource loanPolicy = loanPoliciesFixture.create(canCirculateRollingPolicy);
+
+    useLoanPolicyAsFallback(loanPolicy.getId(),
+        requestPoliciesFixture.allowAllRequestPolicy().getId(),
+        noticePoliciesFixture.activeNotice().getId());
+
+    requestsFixture.placeHoldShelfRequest(smallAngryPlanet, jessica,
+        DateTime.now(DateTimeZone.UTC), requestServicePoint.getId(), "Page");
+
+    requestsFixture.placeHoldShelfRequest(smallAngryPlanet, james,
+        DateTime.now(DateTimeZone.UTC), requestServicePoint.getId(), "Recall");
+
+    final IndividualResource loan = loansFixture.checkOutByBarcode(
+      smallAngryPlanet, jessica, ClockManager.getClockManager().getDateTime());
+
+    final String originalDueDate = loan.getJson().getString("dueDate");
+
+    JsonObject storedLoan = loansStorageClient.getById(loan.getId()).getJson();
+
+    final String recalledDueDate = storedLoan.getString("dueDate");
+    assertThat("due date after recall is not the original date",
+        recalledDueDate, is(originalDueDate));
+
+    ClockManager.getClockManager().setClock(Clock.offset(clock, Duration.ofDays(1)));
+
+    requestsFixture.placeHoldShelfRequest(smallAngryPlanet, charlotte,
+        DateTime.now(DateTimeZone.UTC), requestServicePoint.getId(), "Recall");
+
+    storedLoan = loansStorageClient.getById(loan.getId()).getJson();
+    assertThat("second recall changed the due date (2 weeks)",
+        storedLoan.getString("dueDate"), is(recalledDueDate));
+  }
+
+  @Test
+  public void pagedItemRecalledThenLoanedBecomesOverdueAndNextRecallDoesNotChangeDueDate()
```

Comment:
Why is becoming overdue relevant to this logic?

Reply:
It probably isn't, but it was specified in the scenarios list in CIRC-440, so I wrote a test for it, just to be safe.

Reply:
Ok, that makes sense, if it is important to the process owner, it seems reasonable to explicitly test it